src/ui/components/connect/listConnections/listConnections.html

```diff
@@ -4,7 +4,7 @@
   </div>
   <ul class="connection-list">
     <li ng-repeat="connection in connections" ng-class="{ 'active' : connection.selected }" ng-show="connections.length">
-      <a class="name" href="#" ng-click="selectConnection(connection, $event)">
+      <a class="name" href="#" ng-click="selectConnection(connection, $event)" ng-dblclick="selectConnection(connection, $event);connect($event)">
```

Comment:
So, remove the 'Connect' button?

I really made this change just so it was quicker to get into a query screen and test, didn't mean for it to make it in but I do kind of like it...

I think, instead of a sliding in check mark when you select a connection, we should put a blue highlight around the selected connection. That way you don't have that weird half second of something sliding in before you connect.